uint8 robot_color       # 颜色 (0 红 / 1 蓝)
uint8 task_mode         # 识别模式  (0 自瞄 / 1 大小符)
uint8 visual_valid      # 视觉有效位 (0/1)
uint8 direction         # 拓展装甲板 (0-3)
uint8 bullet_level      # 弹速等级 1 2 3级
float32 robot_pitch     # 欧拉角(度)
float32 robot_yaw       # 欧拉角(度)
float32 time_stamp      # 电控时间戳(ms) 